RecoLocalCalo/HcalRecProducers/python/HcalHitReconstructor_hbhe_cfi.py

```diff
@@ -31,7 +31,9 @@
     setTimingShapedCutsFlags  = cms.bool(True),
     setTimingTrustFlags       = cms.bool(False), # timing flags currently only implemented for HF
     setPulseShapeFlags        = cms.bool(True),
-    setNegativeFlags          = cms.bool(True), # only in HBHE
```

Comment:
Yes, I understood that. I also prefer to leave it as it is -- it is to be changed anyway when the db starts working as expected.